src/main/java/org/molgenis/downloader/api/metadata/Tag.java

```diff
@@ -61,29 +72,30 @@ public void setLabel(String label) {
 
   @Override
   public boolean equals(Object o) {
-    if (this == o) return true;
-    if (!(o instanceof Tag)) return false;
-
-    Tag tag = (Tag) o;
-
-    if (label != null ? !label.equals(tag.label) : tag.label != null) return false;
-    if (objectIRI != null ? !objectIRI.equals(tag.objectIRI) : tag.objectIRI != null) return false;
-    if (relationIRI != null ? !relationIRI.equals(tag.relationIRI) : tag.relationIRI != null)
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
       return false;
-    if (relationLabel != null
-        ? !relationLabel.equals(tag.relationLabel)
-        : tag.relationLabel != null) return false;
-    return codeSystem != null ? codeSystem.equals(tag.codeSystem) : tag.codeSystem == null;
+    }
+    Tag tag = (Tag) o;
+    return Objects.equals(getLabel(), tag.getLabel())
```

Comment:
looks clean :)!

Reply:
It's the new way IntelliJ auto-generates the `equals` and `hashCode` methods 👌